app/src/main/res/layout-sw600dp/toolbar.xml

```diff
@@ -18,7 +18,6 @@
         android:layout_marginTop="8dp"
         android:background="?android:attr/selectableItemBackgroundBorderless"
         android:padding="4dp"
-        android:contentDescription="@string/menu_back"
```

Comment:
Let's add a string here. Even though if we do not do an export for v1. We definitely want to have it in the next time. :)